src/snmalloc/backend_helpers/pagemap.h

```diff
@@ -325,4 +325,81 @@ namespace snmalloc
       body[p >> SHIFT] = t;
     }
   };
+
+  template<
+    typename PAL,
+    typename ConcreteMapT,
+    typename EntryT,
```

Comment:
We don't use `T` suffixes for types anywhere else.  Names that start with upper-case letters are types.